<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="17072" started-at="2016-04-11T20:40:22Z" finished-at="2016-04-11T20:40:39Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="17072" started-at="2016-04-11T20:40:22Z" finished-at="2016-04-11T20:40:39Z">
      <class name="com.globant.training.rocio_bossio.tests.Tests">
        <test-method status="PASS" signature="before()[pri:0, instance:com.globant.training.rocio_bossio.tests.Tests@5be1d0a4]" name="before" is-config="true" duration-ms="1920" started-at="2016-04-11T17:40:22Z" finished-at="2016-04-11T17:40:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method status="PASS" signature="case1()[pri:0, instance:com.globant.training.rocio_bossio.tests.Tests@5be1d0a4]" name="case1" duration-ms="23" started-at="2016-04-11T17:40:23Z" description="Verifing if title displayed is the expected one" finished-at="2016-04-11T17:40:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- case1 -->
        <test-method status="PASS" signature="before()[pri:0, instance:com.globant.training.rocio_bossio.tests.Tests@5be1d0a4]" name="before" is-config="true" duration-ms="1548" started-at="2016-04-11T17:40:23Z" finished-at="2016-04-11T17:40:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method status="PASS" signature="case2()[pri:0, instance:com.globant.training.rocio_bossio.tests.Tests@5be1d0a4]" name="case2" duration-ms="3249" started-at="2016-04-11T17:40:25Z" description="Verifying if the query has results displayed" finished-at="2016-04-11T17:40:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- case2 -->
        <test-method status="PASS" signature="before()[pri:0, instance:com.globant.training.rocio_bossio.tests.Tests@5be1d0a4]" name="before" is-config="true" duration-ms="1569" started-at="2016-04-11T17:40:28Z" finished-at="2016-04-11T17:40:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method status="FAIL" signature="case3()[pri:0, instance:com.globant.training.rocio_bossio.tests.Tests@5be1d0a4]" name="case3" duration-ms="3170" started-at="2016-04-11T17:40:30Z" description="Verifying if a post date is the expected one" finished-at="2016-04-11T17:40:33Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Title is not displayed as expected expected [October 10, 2013] but found [null]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Title is not displayed as expected expected [October 10, 2013] but found [null]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:130)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at com.globant.training.rocio_bossio.tests.Tests.case3(Tests.java:89)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- case3 -->
        <test-method status="PASS" signature="before()[pri:0, instance:com.globant.training.rocio_bossio.tests.Tests@5be1d0a4]" name="before" is-config="true" duration-ms="1504" started-at="2016-04-11T17:40:33Z" finished-at="2016-04-11T17:40:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method status="FAIL" signature="case4()[pri:0, instance:com.globant.training.rocio_bossio.tests.Tests@5be1d0a4]" name="case4" duration-ms="4040" started-at="2016-04-11T17:40:35Z" description="Send form and verify if it was properly sent" finished-at="2016-04-11T17:40:39Z">
          <exception class="org.openqa.selenium.ElementNotVisibleException">
            <message>
              <![CDATA[Element is not currently visible and so may not be interacted with
Command duration or timeout: 13 milliseconds
Build info: version: '2.52.0', revision: '4c2593cfc3689a7fcd7be52549167e5ccc93ad28', time: '2016-02-11 11:22:43'
System info: host: 'GLB-D0173', ip: '10.210.165.229', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=43.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 3ecedeaf-3b2b-4b4e-8fe4-efe8ef831d43]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.ElementNotVisibleException: Element is not currently visible and so may not be interacted with
Command duration or timeout: 13 milliseconds
Build info: version: '2.52.0', revision: '4c2593cfc3689a7fcd7be52549167e5ccc93ad28', time: '2016-02-11 11:22:43'
System info: host: 'GLB-D0173', ip: '10.210.165.229', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_25'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=43.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 3ecedeaf-3b2b-4b4e-8fe4-efe8ef831d43
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:327)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
	at com.sun.proxy.$Proxy6.click(Unknown Source)
	at com.globant.training.rocio_bossio.pages.ContactUsPage.completeForm(ContactUsPage.java:35)
	at com.globant.training.rocio_bossio.tests.Tests.case4(Tests.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
Caused by: org.openqa.selenium.ElementNotVisibleException: Element is not currently visible and so may not be interacted with
Build info: version: '2.52.0', revision: '4c2593cfc3689a7fcd7be52549167e5ccc93ad28', time: '2016-02-11 11:22:43'
System info: host: 'GLB-D0173', ip: '10.210.165.229', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_25'
Driver info: driver.version: unknown
	at <anonymous class>.fxdriver.preconditions.visible(file:///C:/Users/ROCIO~1.BOS/AppData/Local/Temp/anonymous6090049396810302641webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:10045)
	at <anonymous class>.DelayedCommand.prototype.checkPreconditions_(file:///C:/Users/ROCIO~1.BOS/AppData/Local/Temp/anonymous6090049396810302641webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12597)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///C:/Users/ROCIO~1.BOS/AppData/Local/Temp/anonymous6090049396810302641webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12614)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_(file:///C:/Users/ROCIO~1.BOS/AppData/Local/Temp/anonymous6090049396810302641webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12619)
	at <anonymous class>.DelayedCommand.prototype.execute/<(file:///C:/Users/ROCIO~1.BOS/AppData/Local/Temp/anonymous6090049396810302641webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12561)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.ElementNotVisibleException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- case4 -->
      </class> <!-- com.globant.training.rocio_bossio.tests.Tests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
